src/NLog.StructuredLogging.Json.Tests/LoggingException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    [Serializable]
```

Comment:
If it work's it's probably fine then. I think I just remembered seeing it in a lot of other code that was ported with a ```#if ...``` around them.